src/main/java/com/thoughtworks/gauge/command/StartCommand.java

```diff
@@ -23,17 +23,20 @@ public void execute() throws Exception {
         StaticScanner staticScanner = new StaticScanner();
         staticScanner.addStepsToRegistry();
         Server server;
-        boolean multithreading = false;
-        int stream = 1;
-        String streamValue = System.getenv(STREAMS_COUNT_ENV);
-        if (streamValue != null && !streamValue.isEmpty()) {
-            stream = Integer.parseInt(streamValue);
-            multithreading = true;
-            Logger.debug("assuming enable_multithreading=true, because " + STREAMS_COUNT_ENV + " is set to " + streamValue);
+        boolean multithreading = System.getenv(ENABLE_MULTITHREADING_ENV).toLowerCase() == "true";
```

Comment:
Please use https://docs.oracle.com/en/java/javase/14/docs/api/java.base/java/lang/Boolean.html#valueOf(java.lang.String)

Reply:
fixed in later commit